packages/fxa-settings/src/pages/Signup/index.tsx

```diff
@@ -250,9 +251,9 @@ export const Signup = ({
             selectedNewsletterSlugs,
             keyFetchToken: data.SignUp.keyFetchToken,
             unwrapBKey: data.unwrapBKey,
-            // Sync desktop sends a web channel message up on Signup
-            // while Sync mobile does on confirm signup
-            ...(isSyncMobileWebChannel && {
+            // Sync desktop v3 sends a web channel message up on Signup
+            // while OAuth Sync does on confirm signup
```

Comment:
I'm realizing for this to be fully functional, I'm probably going to want to get this done: https://mozilla-hub.atlassian.net/browse/FXA-8905

I might do that separately from this PR (hopefully this sprint) since this PR feels g2g otherwise.